### PR TITLE
Add Druid DB icon

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

With https://github.com/lyft/amundsendatabuilder/pull/257, we are able to index Druid DB into the graph. Add icon class.

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes, including screenshots of any UI changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

File: amundsen_application/static/css/_icons.scss [hunk 1]
```diff
@@ -86,6 +86,11 @@ img.icon {
     background-color: transparent;
   }
 
+  &.icon-druid {
+    content: url('/static/images/icons/logo-druid.svg');
+    background-color: transparent;
+  }
```
Comment (danwom) [inline amundsen_application/static/css/_icons.scss:92]: @ttannis Just wondering why some icons use `content` and others user `mask-image`.

Comment (ttannis) [reply amundsen_application/static/css/_icons.scss:92]: I can't remember the exact reason but our current logos are mix are logos that were generated by our previous contract designers in Sketch, and others that we found freely on the web. I recall some of the ones generated from Sketch worked differently for some reason and I think that's where `mask-image` came in.

**Comment by ttannis** [comment]
@feng-tao will we be showing Druid resources internally? If so, you'll want to update `config-custom` in our private repo with the same lines you added here in `config-dafult`.

**Comment by feng-tao** [comment]
thanks for the pointer, will add that config there as well.

**Pull request merged**